### ISSUE TITLE
Defect Hunting for Sprint 54 - Los Angeles

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#2089]
Please record the issues found during the defect hunting as a thread to this issue.

**Comment by mikerodonnell89** [comment]
Would be great to get this fixed as the ngx fix is coming next sprint: https://github.com/SAP/fundamental-styles/issues/2083

**Comment by mikerodonnell89** [comment]
- [ ] Split button focus weirdness  ![Screen Shot 2021-01-26 at 5 12 24 PM](https://user-images.githubusercontent.com/2471874/105923485-aa160f80-5ff9-11eb-82fe-dc6956dbdf7f.png)

**Comment by JKMarkowski** [comment]
Initial Focus Trap doesn't work on popover 
![Peek 2021-01-27 12-28](https://user-images.githubusercontent.com/26483208/105985009-3298bc80-609b-11eb-930a-b61e1ba523f9.gif)

**Comment by mikerodonnell89** [comment]
- [ ] Floating footer bar should have a z-index - in this example the input is placed above the footer - Mike
![Screen Shot 2021-01-27 at 3 14 49 PM](https://user-images.githubusercontent.com/2471874/106061259-66cba780-60b2-11eb-979f-fd9c5fe15ad7.png)
